### ISSUE TITLE
Choose where to save the map files

### ISSUE THREAD

**Issue by junjunguo** [junjunguo/PocketMaps#46]
user can define where to save the map files (or use default location).

**Comment by hamlet** [comment]
+1
The default location on my phone does not have enough spacei !

**Comment by white-gecko** [comment]
Just a question: what is the default location?

**Comment by stephan75** [comment]
Indeed, users can choose path where map files are downloaded and stored, but there is no indicator what path is set so far.

@Starcommander  ... Can you add radio buttons to the list of paths where user can choose from?

**Comment by Starcommander** [comment]
For sure this should be indicated.
